### PR TITLE
Add airbyte case study

### PR DESCRIPTION
**Pull request by rylandg**
Airbyte case study

### CODE REVIEW

File: blog/2021-08-11-airbyte-case-study.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+tags:
+  - Airbyte
+  - Data pipeline
+  - Machine learning
+  - microservice-orchestration
+  - case study
+posted_on_: 2021-08-11T23:06:09Z
+slug: airbyte-case-study
+title: 'Durable, scalable EL(T) with MIT OSS Airbyte and Temporal'
```
Comment (sw-yx) [inline blog/2021-08-11-airbyte-case-study.md:10]: really want to go with the "MIT OSS"? too many acronyms in a title?

File: blog/2021-08-11-airbyte-case-study.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+release_version: V1.11.3
+---
+
+<img class="case-study-header" src='https://i.imgur.com/ezimJOr.png' />
+
+[Link to PDF version](http://temporal.io/case-studies/airbyte-case-study.pdf)
+
+<!--truncate-->
+
+This case study is based on an interview with [Abhi Vaidyanatha](https://www.linkedin.com/in/avaidyanatha/) and [Jared Rhizor](https://www.linkedin.com/in/jrhizor/) who were key players in Temporal adoption within Airbyte
```
Comment (sw-yx) [inline blog/2021-08-11-airbyte-case-study.md:24]:
```suggestion
*This case study is based on an interview with [Abhi Vaidyanatha](https://www.linkedin.com/in/avaidyanatha/) and [Jared Rhizor](https://www.linkedin.com/in/jrhizor/) who were key players in Temporal adoption within Airbyte.*
```

File: blog/2021-08-11-airbyte-case-study.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+### Context
+
+Data warehouses were originally designed around ETL (extract transform load). With ETL, data is extracted from a source and then transformed to meet the schema of the destination before being loaded (schema on write) but it also adds immense overhead of transforming data before it can be stored. For most of computing history, disk space and cycles were expensive, which made ETL a great choice.
+
+Storage and compute costs have dropped dramatically in the last 20 years, which in turn means the primary argument for ETL is far less relevant. This led to the rise in popularity of an alternative approach known as ELT (extract load transform). Transformation is still required but it becomes the responsibility of the data lake end user (schema on read). During the ETL era, most relevant solutions aimed to address the entire problem even though transforming is a much different problem than extracting and loading. With ELT, the equation changed and extracting and loading are far more isolated from transformations. This opened up the potential for systems which only concern themselves with extracting and loading, and left transformation to a dedicated solution.
```
Comment (sw-yx) [inline blog/2021-08-11-airbyte-case-study.md:30]:
```suggestion
Storage and compute costs have dropped dramatically in the last 20 years, which in turn means the primary argument for ETL is far less relevant. This led to the rise in popularity of an alternative approach known as ELT (extract load transform). Transformation is still required but it becomes the responsibility of the data lake end user (schema on read). During the ETL era, most relevant solutions aimed to address the entire problem even though transforming is a much different problem than extracting and loading. With ELT, extracting and loading are far more isolated from transformations. This opened up the potential for systems which only concern themselves with extracting and loading, and left transformation to a dedicated solution.
```

**Review by sw-yx — APPROVED** [review]

File: blog/2021-08-11-airbyte-case-study.md (hunk 1 above)
Comment (rylandg) [inline blog/2021-08-11-airbyte-case-study.md:10]:
```suggestion
title: 'Durable, scalable EL(T) with OSS Airbyte and Temporal'
```

Comment (rylandg) [reply blog/2021-08-11-airbyte-case-study.md]: Changed

**Pull request merged**